.travis.yml

```diff
@@ -6,7 +6,7 @@ env:
 
 # keep this blank to make sure there are no before_install steps
 before_install:
-
+  npm install codecov
```

Comment:
Why does this need to run before install? Does this not work as a regular dev dependency?